### PR TITLE
doc: known_issues: update USB known issues for 1.4.99

### PR DESCRIPTION
**Pull request by jfischer-no**
Add new known issue for USB.

### CODE REVIEW

File: doc/nrf/known_issues.rst [hunk 1]
```diff
@@ -587,6 +587,15 @@ MCUboot
 Recovery with the USB does not work
   The MCUboot recovery feature using the USB interface does not work.
 
+USB
```
Comment (ru-fu) [inline doc/nrf/known_issues.rst:590]: Should this be a new section, or should it go in the Zephyr section?

Comment (jfischer-no) [reply doc/nrf/known_issues.rst:590]: Zephyr section, will resubmit.

File: doc/nrf/known_issues.rst [hunk 2]
```diff
@@ -587,6 +587,15 @@ MCUboot
...
+***
+
+.. rst-class:: v1-4-99-dev1
+
+Disabling USB causes crash
```
Comment (ru-fu) [inline doc/nrf/known_issues.rst:595]: Is there a Jira issue for this?

File: doc/nrf/known_issues.rst [hunk 3]
```diff
@@ -587,6 +587,15 @@ MCUboot
...
+  Disabling USB by calling usb_disable(), and subsequent call
+  to usb_dc_detach(), tries to free unallocated memory and crashes.
```
Comment (ru-fu) [inline doc/nrf/known_issues.rst:597]:
```suggestion
  Disabling USB by calling :c:func:`usb_disable` with a subsequent call to :c:func:`usb_dc_detach` tries to free unallocated memory and crashes.
```
No linebreaks within a sentence.

File: doc/nrf/known_issues.rst [hunk 4]
```diff
@@ -587,6 +587,15 @@ MCUboot
...
+
```
Comment (ru-fu) [inline doc/nrf/known_issues.rst:598]: Is there some workaround?

**Review by ru-fu — CHANGES_REQUESTED** [review]

File: doc/nrf/known_issues.rst (hunk 2 above)
Comment (jfischer-no) [reply doc/nrf/known_issues.rst]: No, it is fixed upstream, but it missed v1.4.99-dev1, https://github.com/nrfconnect/sdk-nrf/pull/3460

File: doc/nrf/known_issues.rst (hunk 4 above)
Comment (jfischer-no) [reply doc/nrf/known_issues.rst]: No, unfortunately not.

File: doc/nrf/known_issues.rst [hunk 5]
```diff
@@ -912,6 +912,11 @@ NCSDK-6832: SMP Server sample fails upon initialization
 
   **Workaround:** Set :option:`CONFIG_MAIN_STACK_SIZE` to ``2048``.
 
+.. rst-class:: v1-4-99-dev1
```
Comment (ru-fu) [inline doc/nrf/known_issues.rst:915]: Can you move it to the top of the section (since it's new)?

Comment (jfischer-no) [reply doc/nrf/known_issues.rst]: moved to the top

**Review by ru-fu — APPROVED** [review]

**Review by ioannisg — APPROVED** [review]

**Pull request merged**